FSJ_django20_project/FSJ/models_yearofstudy.py

```diff
@@ -0,0 +1,27 @@
+from django.utils.translation import gettext_lazy as _
+from django.db import models
+
+class YearOfStudy(models.Model):
+	#Choices of year for year of study
+	FIRST = 1
+	SECOND = 2
+	THIRD = 3
+	FOURTH = 4
+	FIFTH = 5
+	YEAR_CHOICES = (
+		(FIRST, _('First')),
+		(SECOND, _('Second')),
+		(THIRD, _('Third')),
+		(FOURTH, _('Fourth')),
+		(FIFTH, _('Fifth'))
+	)
+	year = models.IntegerField(choices = YEAR_CHOICES, verbose_name = _("Year of Study"))
```

Comment:
Unable to create new years of study outside of these specific 5. No instances of a year of study exists on fresh install, so new objects need to be created but only allowed to be one of these 5 options. Can create multiple instance of year of study objects with the same year. Does not really allow flexibility of creating additionally years of study beyond hard coded values.